firmware/components/driver_display_eink/include/driver_eink.h

```diff
@@ -1,21 +1,21 @@
 #ifndef DRIVER_EINK_H
 #define DRIVER_EINK_H
 
-#include <stdint.h>
 #include <esp_err.h>
+#include <stdint.h>
 
 #include "driver_eink_types.h"
 
 /** the width of the eink display */
-#define DRIVER_EINK_WIDTH  296
+#define DRIVER_EINK_WIDTH 296
 
 /** the height of the eink display */
 #define DRIVER_EINK_HEIGHT 128
 
 /** the max number of layers used for greyscale */
 #define DRIVER_EINK_MAX_LAYERS 16
 
-#define EINK_BUFFER_SIZE DRIVER_EINK_WIDTH*DRIVER_EINK_HEIGHT
+#define EINK_BUFFER_SIZE DRIVER_EINK_WIDTH* DRIVER_EINK_HEIGHT
 
```

Comment:
This doesn't seem like we want it happening - can the formatter be taught to not do this to constants?

Reply:
Maybe!

Reply:
Looks like it thinks that this is a pointer. Maybe we should just change comments like this to have parentheses? Would even prevent some errors...

Reply:
yes, adding parentheses around this would be nicer, agreed.

Reply:
Should I add those now, or do we get whoever fiddles with the files next to do that, respectively?

There's a few warnings I saw in travis that I'd like to address, so I could just lump all that stuff together in a code quality PR.